Kickstarter-iOS/DataSources/ProjectSummaryCarouselDataSource.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import KsApi
+import Library
+import UIKit
+
+internal final class ProjectSummaryCarouselDataSource: ValueCellDataSource {
+  public private(set) var greatestCombinedTextHeight: CGFloat = 0
+
+  internal enum Section: Int {
+    case summary
+  }
+
+  func load(_ values: [ProjectSummaryItem]) {
+    // TODO: iterate over values, calculate greatest item height and cache it
```

Comment:
We want all of the cells to be the size of the largest one so we will do this at each reload. If dynamic type changes occur, we will also reload the datasource and collection view for simplicity.